test/run-tests

```diff
@@ -1140,7 +1163,7 @@ def main():
         pattern_matched = False
         for pattern in image_patterns:
             if fnmatch.fnmatch(image["name"], pattern) or fnmatch.fnmatch(
-                image["source"], pattern
+                image.get("source", ""), pattern
```

Comment:
should we try to match on source or compose here?  I suppose the current code doesn't handle openstack images anyway

Reply:
Do we use it to match on anything else than name in practice?

Reply:
Probably not - I would say this is ok - and if a dev needs to do something else with pattern (e.g. for debugging) that dev can enhance the code to do the needful